Lib/importlib/abc.py

```diff
@@ -22,6 +23,11 @@
 ]
 
 
+def __getattr__(name):
+    if name in ('ResourceLoader' ):
```

Comment:
If you meant to use a tuple, you need the trailing comma
```suggestion
    if name in ('ResourceLoader',):
```
Or since it's just a single name, use normal comparison:
```suggestion
    if name == 'ResourceLoader':
```